releasenotes/releasenotes_v5.rst

```diff
@@ -5,8 +5,129 @@ Current Firely Server release notes (v5.x)
+#. You can now add signatures to ``AuditEvents``. See :ref:`audit_event_integrity` for more information.
+#. Firely Server now supports searching on version-specific references. Consult the `FHIR specification <https://www.hl7.org/fhir/search.html#versions>`_ for more information.
+#. Serilog CorrelationId support has been enabled in Firely Server. Please consult the `official documentation <https://github.com/ekmsystems/serilog-enrichers-correlation-id>`_ on how to configure it.
+#. We have added a public :ref:`Postman collection <postman_tutorial>` to test Firely Server's RESTful endpoints.
+#. Wildcard support for ``include`` is now declared in Firely Server's ``CapabilityStatement``.
+
+Fix
+^^^
+
+#. When performing a Bulk Data Export request with a Firely Server instance running on a SQL database, it will return the Group resource even if it has no members. 
+#. FS now declares support for Bulk Data Export Group export operations in its CapabilityStatement. This features was available before, but missing from FS's CapabilityStatement. 
+#. Bulk Data Export now returns a succesful status code (``202``) instead of an erroneous status code if no resources were matched for an export. The resulting export will include an empty array as described in the `specification <https://hl7.org/fhir/uv/bulkdata/export/index.html#response---complete-status>`_.
+#. Upon commencing a Bulk Data Export, Firely Server now correctly handles ``Prefer`` headers as outlined `in the specification <https://hl7.org/fhir/uv/bulkdata/export/index.html#headers>`_.
+#. ``Device`` can now be added as an additional resource in a Bulk Data export.
+#. Search parameters without a value are now ignored by the server instead of resulting in an error response.
+#. Firely Server now creates valid FHIR R5 AuditEvents.
+#. Searching for a resource with multiple sort fields does not throw an exception anymore when Firely Server runs on a SQL database.
+#. When using the ``If-Modified-Since`` header, only resources that were modified after the specified timestamp are returned. Because of a precision mismatch (seconds vs. milliseconds), wrong resources were sometimes returned before this fix.
+#. When updating a deleted resource conditionally, Firely Server does not throw an exception anymore.
+#. Firely Server now returns the correct issue code (``business-rule`` instead of ``invalid``) in the OperationOutcome when performing a conditional update using ``_id`` as a parameter. Additionally, the error message has been improved when a resource in a different information model is matched via the ``id`` field.
+#. When executing a ``POST``-based search, Firely Server will now return the correct self-link as seen in ``GET``-based searches.
+#. The client id of the default SMART authorization options have been changed from ``vonk`` to ``firelyserver``.
+#. Firely Server now returns improved error messages if the client is not allowed to perform searches.
```

Comment:
... due to insufficient SMART v2 scopes.

Reply:
I feel like the sentence holds true on its own but your version is more precise, so I added it.